simulator/source/Combat_simulator.cpp

```diff
@@ -1476,7 +1537,7 @@ void Combat_simulator::normal_phase(Sim_state& state, bool mh_swing)
         {
             use_sa &= time_keeper_.whirlwind_cd() > config.combat.sunder_armor_cd_thresh;
         }
-        if (rage > config.combat.sunder_armor_rage_thresh && rage >= 15 && use_sa)
+        if (rage > config.combat.sunder_armor_rage_thresh && rage >= sunder_armor_rage_cost_ && use_sa)
         {
             sunder_armor(state);
             return;
```

Comment:
This is the recently added "prefer building 5 sunder stacks" option: I'd add devastate here similar to how it's added for the "sunder_first_globals" case, simply replacing sunder_armor() if it's available. Re-using the cd- and rage-thresholds is fine here, I guess.

Reply:
I'm stupid for missing this! I think we're bound to have a weird issue where the "use sunder armor" option to stack sunders logically conflicts with the "use devastate" option for doing damage in the case that both are checked, but since it's not really all that useful to configure specific options in "use sunder" when you're using devastate almost every GCD I don't think it makes sense to mess with the thresholds here for the sunder_armor_rage_thresh and sunder_armor_rage_cost_ and try to use different ones related to devastate since they should never really be different.